dashboard/test/controllers/new_feature_feedback_controller_test.rb

```diff
@@ -0,0 +1,66 @@
+require 'test_helper'
+
+class NewFeatureFeedbackControllerTest < ActionController::TestCase
+  self.use_transactional_test_case = true
+
+  setup_all do
+    @user = create(:teacher)
+  end
+
+  setup do
+    sign_in @user
+  end
+
+  teardown do
+    NewFeatureFeedback.where(user: @user).destroy_all
+  end
+
+  test 'create - creates user New Feature Feedback' do
+    expected_satisfied = true
+    expected_form_key = 'progress_v2'
+
+    assert_difference('NewFeatureFeedback.count') do
+      post :create, params: {feedback: {satisfied: expected_satisfied, form_key: expected_form_key}}, format: :json
+    end
+
+    assert_response :created
+    response = JSON.parse(@response.body)
+    assert_equal @user.id, response['user_id']
+    assert_equal expected_satisfied, response['satisfied']
+    assert_equal expected_form_key, response['form_key']
+  end
+
+  test 'create - returns validation errors when satisfied param is invalid' do
+    assert_no_difference('NewFeatureFeedback.count') do
+      post :create, params: {feedback: {satisfied: nil, form_key: 'progress_v2'}}, format: :json
+    end
+
+    assert_response :unprocessable_entity
+    assert_equal '["Satisfied is not included in the list"]', response.body
+  end
+
```

Comment:
A test to ensure users can't leave feedback on the same thing twice would be nice.